src/kademlia/node.cpp

```diff
@@ -695,18 +652,18 @@ void node::lookup_peers(sha1_hash const& info_hash, entry& reply
 	}
 	else
 	{
-		int num = (std::min)(int(v.peers.size()), m_settings.max_peers_reply);
-		std::set<peer_entry>::const_iterator iter = v.peers.begin();
+		int num = (std::min)(int(v_peers.size()), m_settings.max_peers_reply);
+		std::vector<std::pair<tcp::endpoint, bool> >::const_iterator iter = v_peers.begin();
```

Comment:
maybe you can add a comment somewhere indicating that the bool here indicates whether the peer is a seed or not.